parse/table_definitions_arbitrum/circle/TokenMessenger_event_DepositForBurn.json

```diff
@@ -0,0 +1,109 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "uint64",
+                    "name": "nonce",
+                    "type": "uint64"
+                },
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "burnToken",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "amount",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "depositor",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "bytes32",
+                    "name": "mintRecipient",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint32",
+                    "name": "destinationDomain",
+                    "type": "uint32"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "bytes32",
+                    "name": "destinationTokenMessenger",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "bytes32",
+                    "name": "destinationCaller",
+                    "type": "bytes32"
+                }
+            ],
+            "name": "DepositForBurn",
+            "type": "event"
+        },
+        "contract_address": "0x19330d10d9cc8751218eaf51e8885d058642e08a",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "arbitrum_circle",
```

Comment:
I would just use "circle" as dataset name here, the "arbitrum_" will be added as a suffix automatically. Same for the remaining files.